events/urls.py

```diff
@@ -10,4 +10,6 @@
     path("delete/<int:event_id>/", views.deleteEvent, name="delete-event"),
     path("update/<int:event_id>/", views.updateEvent, name="update-event"),
     path("<int:event_id>/", views.eventDetail, name="event-detail"),
+    path("get_data/", views.get_data, name="get_data"),
```

Comment:
we should not have _ in url
can you replace this with '-'

Reply:
Also we dont need verbs in url
You can change it to locations/ and the httpget will signify that you want to get locations

Reply:
Okay will make the change

Reply:
Change has been made. Please take a look